src/Jackett.Common/Indexers/DarmoweTorrenty.cs

```diff
@@ -0,0 +1,234 @@
+            var dateMatch = _DateRegex.Match(detailsRow.TextContent);
+            var sizeMatch = _SizeRegex.Match(detailsRow.TextContent);
+            var date = DateTime.MinValue; // In case of parsing failure
+            if (dateMatch.Success)
+            {
+                var polishDate = DateTime.SpecifyKind(DateTime.ParseExact(dateMatch.Groups[1].Value, "dd/MM/yyyy", CultureInfo.InvariantCulture), DateTimeKind.Unspecified);
+                var pubDateUtc = TimeZoneInfo.ConvertTimeToUtc(polishDate, _polandTz);
+                date = pubDateUtc.ToLocalTime();
+            }
+            var details = titleRow.QuerySelector("a[href^=\"details.php?id=\"]:has(span)");
+            var detailsLink = new Uri(SiteLink + details.GetAttribute("href"));
+            var encodedDownloadLink = detailsRow.QuerySelector("a[id^=\"download_\"]").GetAttribute("data-href");
+            var downloadLink = new Uri(SiteLink + Uri.UnescapeDataString(StringUtil.FromBase64(encodedDownloadLink)));
+            var bannerLink = detailsRow.QuerySelector("img[src^=\"./imgtorrent/\"]")?.GetAttribute("src");
+            var seeders = seedsMatch.Success ? int.Parse(seedsMatch.Groups[1].Value) : 0;
+            var leechers = leechersMatch.Success ? int.Parse(leechersMatch.Groups[1].Value) : 0;
+            var peers = seeders + leechers;
+            var release = new ReleaseInfo
+            {
+                Title = details.TextContent,
+                Category = categories,
+                Seeders = seeders,
+                BannerUrl = !string.IsNullOrEmpty(bannerLink) ? new Uri(SiteLink + bannerLink) : null,
+                Peers = peers,
+                PublishDate = date,
+                DownloadVolumeFactor = 0,
+                UploadVolumeFactor = 1,
+                Link = downloadLink,
+                Guid = detailsLink,
+                Comments = detailsLink,
+                Size = sizeMatch.Success ? ReleaseInfo.GetBytes(sizeMatch.Groups[1].Value) : 0
+            };
+            return release;
+        }
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var startTransition = TimeZoneInfo.TransitionTime.CreateFloatingDateRule(new DateTime(1, 1, 1, 3, 0, 0), 3, 5, DayOfWeek.Sunday);
+            var endTransition = TimeZoneInfo.TransitionTime.CreateFloatingDateRule(new DateTime(1, 1, 1, 4, 0, 0), 10, 5, DayOfWeek.Sunday);
+            var delta = new TimeSpan(1, 0, 0);
+            var adjustment = TimeZoneInfo.AdjustmentRule.CreateAdjustmentRule(new DateTime(1999, 10, 1), DateTime.MaxValue.Date, delta, startTransition, endTransition);
+            TimeZoneInfo.AdjustmentRule[] adjustments = { adjustment };
+            _polandTz = TimeZoneInfo.CreateCustomTimeZone("C. Europe Standard Time", new TimeSpan(1, 0, 0), "(GMT+01:00) C. Europe Standard Time", "C. Europe Standard Time", "C. Europe DST Time", adjustments);
```

Comment:
This implementation has a lot of date related code but the .yml doesn't. What do you need all of this?

Reply:
Another indexer i've based on was handling date/time as such. I've now changed it to be the same as in yml.